validate-commit-msg-bot.md

```diff
@@ -0,0 +1,47 @@
+---
+title: validate-commit-msg-bot
+description: Runs validate-commit-msg over all commits of new or edited pull requests and sets an appropriate a status check.
+slug: validate-commit-msg-bot
+screenshots:
+- https://github.com/tlvince/validate-commit-msg-bot/raw/master/docs/status-check-screenshot.png
+authors: [ tlvince ]
+repository: tlvince/validate-commit-msg-bot
```

Comment:
Any chance you'd consider renaming to `validate-commit-msg`? See https://github.com/probot/probot.github.io/pull/56#discussion_r140643393 for more on why.

Reply:
That makes sense, although I used `-bot` to differentiate from [validate-commit-msg](https://github.com/conventional-changelog-archived-repos/validate-commit-msg) (which does the heavy lifting). It could add confusion. What do you think?

Reply:
I think it will be okay given it's based off the library.